src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2239,7 +2240,7 @@ void processContextElement
       {
         ContextAttribute* ca = new ContextAttribute(aP);
 
-        buildGeneralErrorResponse(ceP, ca, responseP, SccInvalidParameter,
+        buildGeneralErrorResponse(ceP, ca, responseP, SccInvalidModification,
```

Comment:
After a skype discussion, at the end we will keep the code as it is now, due to:

* Backward compabilitiy is really really small
* It involves an improvement
* Keep backward compability infolves extra work
* Kepp backward compability involves "dirty" code due to the "if (apiVersion == v2)"  exceptions.

However, let's make this explicit in the CNR, with an additiona entry like this:

"Fix: chaned error code from 472 (Invalid Parameter) changed to 422 (Invalid Moficiation) in NGSIv1 in the case of missing attribute value (#1164)"